wmi.h

```diff
@@ -0,0 +1,9 @@
+#ifndef __WMI_H__
```

Comment:
Why do we need two separate files with the same name?
There's another wmi.h in WindowsStubs/WindowsStubs .